src/Kripke/Core/Field.h

```diff
@@ -241,7 +244,11 @@ namespace Core {
 
         LType layout = RAJA::make_stride_one<LInfo::stride_one_dim>(m_chunk_to_layout[chunk_id]);
 
-        return ViewType<Order, ElementType, ElementPtr, IDX_TYPES...>(ptr, layout);
+#if defined(KRIPKE_USE_HIP) || defined(KRIPKE_USE_CUDA)
```

Comment:
This was giving me a compilation error at `chai::GPU` in the line below.

```suggestion
#if (defined(KRIPKE_USE_HIP) || defined(KRIPKE_USE_CUDA)) && defined(KRIPKE_USE_CHAI)
```

Reply:
@vsrana01 RAJA Proxy still needs this to work. If this is ok with you, would you mind committing this change? After this, I think we're good to go with merging this branch into develop.